### DISCUSSION TITLE
Reveal Presenations Background Colour Blocks (SCSS)

### DISCUSSION THREAD

**Discussion by maltehueckstaedt** [quarto-dev/quarto-cli#5285] [Q&A]
Hi, I would like to build a theme for a Reveal presentation similar to the one used by my employer. To do this, I would like to split the background of the default slides into three colour blocks: A narrow green vertical bar, a slightly thicker grey vertical bar and then the rest in white. Like this:
https://github.com/maltehueckstaedt/dzhw_revealjs_presentation/blob/main/images/Folie.png?raw=true

I would be super grateful if anyone could help me to prepare my [scss](https://github.com/maltehueckstaedt/dzhw_revealjs_presentation/blob/main/custom.scss) so that the slides appear like on the example above.

Thanks for any help!

**Comment by mcanouil** [comment]
I am sorry, but if your question is purely on SCSS/CSS, there are better places to ask for that, such as StackOverflow or StackExchange for example.
This being said, take a look at Quarto gallery, Quarto extension listing, Awesome Quarto GitHub list.

- https://quarto.org/docs/gallery/#presentations
- https://quarto.org/docs/extensions/listing-revealjs.html
- https://github.com/mcanouil/awesome-quarto#presentations
